.github/workflows/ci.yaml

```diff
@@ -6,13 +6,20 @@ on:
   release:
     types:
       - released
-  pull_request:
+  pull_request_target:
```

Comment:
Since `pull_request_target` runs on `base`, we'll need to first merge it in to check that it works